src/boundingboxes.jl

```diff
@@ -76,7 +76,7 @@ boundingbox(g::RectilinearGrid) = Box(extrema(g)...)
 boundingbox(g::TransformedGrid{Dim,<:CartesianGrid{Dim}}) where {Dim} =
   boundingbox(parent(g)) |> transform(g) |> boundingbox
 
-boundingbox(g::TransformedGrid{Dim,<:RectilinearGrid{Dim}}) where {Dim} =
+boundingbox(g::TransformedGrid{Dim,<:RectilinearGrid{Datum,Dim}}) where {Datum,Dim} =
```

Comment:
I believe that we can omit the type parameters in these methods, since they are not used.